protostar/configuration_file/configuration_file.py

```diff
@@ -86,6 +86,20 @@ def set_command_names_provider(
     ):
         pass
 
+    @staticmethod
+    def create_appending_cairo_path_suggestion() -> str:
+        return "\n".join(
+            [
+                "You may want to add your new library's path to the 'cairo_path' section of your `protostar.toml` file",
+                "[protostar.shared_command_configs]",
+                "cairo_path = [",
+                '    "...",',
+                '    "path/to/the/library/source/files"',
+                "]",
```

Comment:
This config is specific to the ConfigurationFileV1. You can remove the example. The generic message and a link to the documentation should be good enough.

Reply:
I see, alright

Reply:
[address feedback - make the message more generic](https://github.com/software-mansion/protostar/pull/981/commits/7eab993b8be6705e14f2a27944838915254a6ca9)